libc/test/UnitTest/LibcTest.cpp

```diff
@@ -126,13 +115,17 @@ int Test::runTests(const char *TestFilter) {
       continue;
     }
     tlog << GREEN << "[ RUN      ] " << RESET << TestName << '\n';
+#ifdef LIBC_TEST_USE_CLOCK
     [[maybe_unused]] const auto start_time = clock();
```

Comment:
`[[maybe_unused]]` is not necessary anymore if the lines are compiled conditionally.

Reply:
Done.